README.md

```diff
@@ -140,6 +170,6 @@ While it's possible to change the storage to use `Retain` vs `Delete`, the helm
 
 ## Auto Scaling
 
-The worker pods are configured to auto-scale based on CPU threshold (default 50%). Once the aggregate CPU for all worker pods exceed the defined threshold (in this case 50%), the Kubernetes engine will start adding additional worker pods up to the maximum specified. This is also dependent on how the Kuebernetes cluster was configured as additional VM node instances will also be added. Please refer to the notes on [aws](/aws/README.md) and [google](/google/README.md) when setting up the cluster and note the instance type and maximum nodes specified.
+The worker pods are configured to auto-scale based on CPU threshold (default 12%). Once the aggregate CPU for all worker pods exceed the defined threshold (in this case 12%), the Kubernetes engine will start adding additional worker pods up to the maximum specified. This is also dependent on how the Kuebernetes cluster was configured as additional VM node instances will also be added. Please refer to the notes on [aws](/aws/README.md) and [google](/google/README.md) when setting up the cluster and note the instance type and maximum nodes specified.
```

Comment:
why 12% instead of 50%, whats the issue, is it relevant for everyone?

Reply:
This change may not necessarily be needed. We made this change to try to speed up the time it takes to scale out to the maximum number of worker pods. However, now that you changed the code so that it can start the next analysis before the current one is totally finished, this isn't as much of an issue because the scaling up period should only occur one time at the beginning of the job submission instead of at the beginning of each analysis. Therefore, I'm fine with changing it back to 50%.